polls/tests/test_voting.py

```diff
@@ -173,3 +174,24 @@ def test_show_results(self):
         self.assertContains(response, 'Thank you for voting!')
         response = client.get('/')
         self.assertContains(response, 'You voted')
+
+    def test_free_text_vote(self):
+        question = FreeTextQuestion(
+            title='is this a test')
+        self.english.add_child(instance=question)
+        question.save_revision().publish()
+        client = Client()
+        client.login(username='tester', password='tester')
+        response = client.get('/')
+        self.assertContains(response, 'is this a test')
+        client.post(reverse('molo.polls:free_text_vote',
+                    kwargs={'question_id': question.id}),
+                    {'answer': 'this is an answer'})
+        response = client.get(reverse(
+            'molo.polls:results',
+            kwargs={'poll_id': question.id}))
+        votes = FreeTextVote.objects.all().count()
+        self.assertEquals(votes, 1)
+        self.assertContains(response, 'Thank you for voting!')
+        response = client.get('/')
+        self.assertContains(response, 'Thank you!')
```

Comment:
You should also add a test that ensures a user can only vote once for Free Text question